FTPizza.Tests/FTP.Intergration.Test.cs

```diff
@@ -28,6 +28,61 @@ public void ShouldDownloadAFIle()
             Assert.IsTrue(System.IO.File.Exists("512KB.zip"));
         }
 
+        [Test]
+        public void LoginToFtpWithBadLogin()
+        {
+            // TODO: Needs a more specific exception
+            Assert.That(() => new Ftp(null, null, null), Throws.Exception);
+        }
+
+        [Test]
+        public void SuccessfulLoginIntoFtpServer()
+        {
+            var client = new Ftp("", "", "speedtest.tele2.net/");
+
+            Assert.That(client, Is.Not.Null);
+        }
+
+        [Test]
+        public void SuccessfulPutFileInToFtpServer()
+        {
+            var upload_client = new Ftp("", "", "speedtest.tele2.net/upload");
+            List<string> uploadList = new List<string>();
+
+            // Make sure file UnitTestFile.txt exists locally in FTPizza\FTPizza.Tests\bin\Debug before executing tests
```

Comment:
Maybe call a streamwriter here to create the file or create one during Setup attribute